servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherFlatMapConcatUtils.java

```diff
@@ -134,31 +136,27 @@ private void tryPollQueue() {
         SingleSource.Subscriber<? super R> subscriber;
         @Nullable
         private Object result;
```

Comment:
Consider adding a comment to clarify why it doesn't have to be `volatile`

Reply:
Yes I also think that this needs to be `volatile`. If not, what guarantees visibility from the producer to the consumer?